### PR TITLE
Fix Committee Links

### PR DESCRIPTION
**Pull request by yeadamk**
## Overview

1. Changed orderings of the committee links
2. Added new discord, instagram, and website links for the following committees:
- ICPC
- Cyber
- AI
- W

## Changes

`data/committees.js`

### CODE REVIEW

File: data/committees.js [hunk 1]
```diff
@@ -161,14 +169,14 @@ export default [
 			// 	platform: 'website',
 			// 	link: 'https://design.uclaacm.com/',
 			// },
-			{
-				platform: 'facebook',
-				link: 'https://www.facebook.com/groups/acmdesign',
-			},
 			{
 				platform: 'instagram',
 				link: 'https://www.instagram.com/acmdesign.ucla',
```
Comment (snigdha-kansal) [inline data/committees.js:174]: seems design's current insta link is outdated, here's the new one: https://www.instagram.com/ucladesign

File: data/committees.js [hunk 2]
```diff
@@ -290,14 +302,14 @@ export default [
...
-				platform: 'facebook',
-				link: 'https://www.facebook.com/teachlaucla/',
-			},
 			{
 				platform: 'instagram',
 				link: 'https://www.instagram.com/acm.teachla/',
 			},
+			{
+				platform: 'facebook',
+				link: 'https://www.facebook.com/teachlaucla/',
```
Comment (snigdha-kansal) [inline data/committees.js:311]: seems it's moved to https://www.facebook.com/acmteachla/

File: data/committees.js (hunk 1 above)
Comment (yeadamk) [reply data/committees.js]: done

File: data/committees.js (hunk 2 above)
Comment (yeadamk) [reply data/committees.js]: done

File: data/committees.js [hunk 3]
```diff
@@ -421,13 +437,17 @@ export default [
...
 			// },
 			{
-				platform: 'facebook',
-				link: 'https://www.facebook.com/groups/uclaacmai',
+				platform: 'discord',
+				link: 'https://discord.gg/QcU9AVW9hm',
 			},
 			{
 				platform: 'instagram',
 				link: 'https://www.instagram.com/acm_ai_ucla/',
```
Comment (snigdha-kansal) [inline data/committees.js:445]: seems it's moved to https://www.instagram.com/acm.ai.ucla

File: data/committees.js [hunk 4]
```diff
@@ -497,14 +517,14 @@ export default [
 				platform: 'website',
 				link: 'https://cloud.uclaacm.com/',
 			},
-			{
-				platform: 'instagram',
-				link: 'https://www.instagram.com/acmcloud/',
-			},
 			{
 				platform: 'discord',
 				link: 'https://discord.gg/smE3pk5DsJ',
```
Comment (snigdha-kansal) [inline data/committees.js:522]: looks like this one expired - i've asked cloud for a new one that doesn't expire

Comment (snigdha-kansal) [reply data/committees.js:522]: https://discord.com/invite/s2VGxKqTYA

File: data/committees.js [hunk 5]
```diff
@@ -162,12 +170,12 @@ export default [
 			// 	link: 'https://design.uclaacm.com/',
 			// },
 			{
```
Comment (snigdha-kansal) [inline data/committees.js:172]: https://discord.gg/hS62MnPm

just got discord link for design - can we add it before instagram? thanks!

**Review by snigdha-kansal — APPROVED** [review]
Looks great to me! Thanks so much for doing this, Yeadam 🔥💯 
Left one last comment, feel free to merge after addressing

Cat Apple

**Pull request merged**